client/client.go

```diff
@@ -1146,10 +1140,7 @@ func (r *NotaryRepository) DeleteTrustData(deleteRemote bool) error {
 	}
 	// Note that this will require admin permission in this NotaryRepository's roundtripper
 	if deleteRemote {
-		remote, err := getRemoteStore(r.baseURL, r.gun, r.roundTrip)
-		if err != nil {
-			return err
-		}
+		remote := r.remoteStore
```

Comment:
Similarly to above, should we check that `r.remoteStore` isn't nil first, and then if it isn't, we can also just call `r.remoteStore.RemoveAll()`?